libr/anal/meson.build

```diff
@@ -34,6 +34,9 @@ r_anal_sources = [
   'var.c',
   'vtable.c',
   'xrefs.c',
+  'passmanager/functionpm.c',
```

Comment:
as i said, this is the only module having a subdirectory that is not d/ or p/, and imho the naming should be libr/anal/pm_fcn.c pm_bin.c pm_bb.c ... otherwise maybe just pm/